JS_lesson4/Task3.js

```diff
@@ -0,0 +1,29 @@
+// Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+const age = Number(prompt(`Введи возраст`));
```

Comment:
No need to convert input to a number, we should check it with provided conditions